lang/fr.json

```diff
@@ -0,0 +1,28 @@
+{
+  "VISION5E.Blindsense": "Perception aveugle",
+  "VISION5E.BlindSenses": "Sens aveugles",
+  "VISION5E.Burrowing": "Fouissage",
+  "VISION5E.DetectEvilAndGood": "Détection du mal et du bien",
+  "VISION5E.DetectMagic": "Détection de la magie",
+  "VISION5E.DetectPoisonAndDisease": "Détection du poison et des maladies",
+  "VISION5E.DetectThoughts": "Détection des pensées",
+  "VISION5E.DevilsSight": "Vision de diable",
+  "VISION5E.DivineSense": "Perception divine",
+  "VISION5E.Echolocation": "Écholocalisation",
+  "VISION5E.Ethereal": "Éthéré",
+  "VISION5E.EtherealSight": "Vision éthérée",
+  "VISION5E.GhostlyGaze": "Regard fantomatique",
+  "VISION5E.Hearing": "Audition",
+  "VISION5E.Inaudible": "Inaudible",
+  "VISION5E.InvocationDevilsSight": "Manifestation : Vision du diable [Occultiste]",
+  "VISION5E.InvocationWitchSight": "Manifestation : Vision sorcière [Occultiste]",
+  "VISION5E.LightPerception": "Perception de la lumière",
+  "VISION5E.MagicAwareness": "Conscience magique",
+  "VISION5E.SeeInvisibility": "Détection de l'invisibilité",
+  "VISION5E.SenseMagic": "Détection de la magie",
+  "VISION5E.Shapechanger": "Métamorphe",
+  "VISION5E.UmbralSight": "Vision des ombres",
+  "VISION5E.WitchSight": "Vision sorcière",
```

Comment:
Seems like "Vision de sorcier" is another translation that is used. Is either more official than the other?

Reply:
The new translation from 2021 is "Vision sorcière". But the old one was "Vision de sorcier". I'll add a pull request in the french translation to uniform the feat.